storage/database/schema.go

```diff
@@ -129,6 +129,8 @@ var (
 	stakingInfoPrefix = []byte("stakingInfo")
 
 	chaindatafetcherCheckpointKey = []byte("chaindatafetcherCheckpoint")
+
+	lastPrunedBlockNumberKey = []byte("lastPrunedBlockNumber")
```

Comment:
Please place it at line 100, below other pruning-related keys.

Reply:
Done.